js/contextMenus.js

```diff
@@ -1093,7 +1093,8 @@ function mainTemplateInit (nodeProps, frame) {
   })
 
   const passwordManager = getActivePasswordManager()
-  if (passwordManager.get('extensionId')) {
+  if (passwordManager.get('extensionId') &&
+    passwordManager.get('displayName') === 'Dashlane') {
```

Comment:
because dashlane doesn't create context menus and other's do which cause repeated entries shows in https://github.com/brave/browser-laptop/issues/5291